src/Text/LLVM/PP.hs

```diff
@@ -980,9 +980,24 @@ ppConstExpr' pp expr =
                        in reverse  -- ppTyp's pushes entries to the listg head
                           $ foldl (ppTyp's mrng) [ppType ty]
                           $ zip argIndices (ptr:ixs)))
-    ConstConv op tv t  -> ppConvOp op <+> parens (ppTyp' tv <+> "to" <+> ppType t)
+    ConstConv op tv t  ->
+      let droppedIn18 = case op of
+                          ZExt _ -> True
+                          SExt -> True
+                          FpTrunc -> True
+                          FpExt -> True
+                          FpToUi -> True
+                          FpToSi -> True
+                          UiToFp _ -> True
+                          SiToFp -> True
+                          _ -> False
+          ppConstConv = ppConvOp op <+> parens (ppTyp' tv <+> "to" <+> ppType t)
+      in if droppedIn18
+         then droppedInLLVM 18 "fptrunc/fpext/fptoui/fptosi/uitofp/sitofp constexprs" ppConstConv
+         else ppConstConv
     ConstSelect c l r  ->
-      "select" <+> parens (commas [ ppTyp' c, ppTyp' l , ppTyp' r])
+      droppedInLLVM 17 "select constexpr"
```

Comment:
This one was removed in https://github.com/llvm/llvm-project/commit/bbfb13a5ffbccf1759ca6b75262a3ffdbe20496e.